helix-core/src/main/java/org/apache/helix/ConfigAccessor.java

```diff
@@ -118,6 +118,11 @@ public ConfigAccessor(HelixZkClient zkClient) {
     _usesExternalZkClient = true;
   }
 
+  public ConfigAccessor(RealmAwareZkClient zkClient) {
+    _zkClient = zkClient;
+    _usesExternalZkClient = true;
+  }
```

Comment:
The new ConfigAccessor should be created with a Builder. I'm going to assume that this is just a placeholder? :)

Reply:
On a second thought, I forgot to add this constructor. I'll go ahead and work on that as well.